modules/dkan/dkan_harvest/README.md

```diff
@@ -43,7 +43,7 @@ Project Open Data (as well as most metadata APIs) includes many fields that are
 To access the name property for filtering or overriding, you can set `publisher.name` in the first text box and the value you want to use in the second one.
 
 
-![Add Harvest Source](images/node_add_harvest_source.png)
+![Add Harvest Source](https://cloud.githubusercontent.com/assets/381224/19936998/53e3012c-a0fe-11e6-8253-16c5270eaedf.png)
```

Comment:
Also, the screenshot of the harvest source page should include a description now